solr/core/src/java/org/apache/solr/search/SolrDocumentFetcher.java

```diff
@@ -539,6 +541,21 @@ public Reader readerValue() {
     public Number numericValue() {
       return null;
     }
+
+    @Override
+    public StoredValue storedValue() {
+      if (fieldType().stored() == false) {
+        return null;
+      } else {
+        return new StoredValue(stringValue());
+      }
+    }
+
+    @Override
+    public InvertableType invertableType() {
+      // TODO SOLR-16799 when to use InvertableType.TOKEN_STREAM and when to use null?
+      return null;
+    }
```

Comment:
It shouldn't matter, right? this is only for stored field retrieval, not for indexing. This should never be called AFAIK

Reply:
hmm. I agree, will remove the TODO